lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -532,8 +532,10 @@ def format_exp_date(month, year)
 
       def parse(kind, xml)
         parsed = {}
-
         doc = Nokogiri::XML(xml).remove_namespaces!
+
+        parsed['duplicate'] = doc.at_xpath('//saleResponse')['duplicate'] == 'true' if kind == :sale
```

Comment:
One worry I have here is that if `doc.at_xpath('//saleResponse')` returns `nil` then calling `['duplicate']` on it will cause ruby to say `NoMethodError: undefined method '[]' for nil:NilClass` and then we are returning a sad 500 to the customer.

It looks like we make assumptions like this elsewhere in the `parse` method, and it could very well be that `saleResponse` is ALWAYS there when `kind == :sale`, but I would still feel better if we protected against a `nil` here. Something like:

```
parsed['duplicate'] = doc.at_xpath('//saleResponse').try(:[], 'duplicate') == 'true' if kind == :sale
```